index.js

```diff
@@ -273,7 +273,7 @@ async.parallelPlus = function(functions, callback) {
     }
     var newFunctions = {};
     for (var func in functions) {
-        if (Object.prototype.hasOwnProperty.call(functions, func)) {
+        if (functions.hasOwnProperty(func)) {
```

Comment:
Since `hasOwnProperty` is on `Object.prototype` anyways, that means it is automatically available on any object who has `Object.prototype` in its `[[Proto]]` chain, functions included.